add-on/chrome/skin/shared/loop.css

```diff
@@ -54,11 +54,17 @@
      the same as the hover state to visually indicate that it's in-use.
      */
   #loop-button[state="slideshow"] {
-    background: var(--toolbarbutton-hover-background);
     border-color: var(--toolbarbutton-hover-bordercolor);
     box-shadow: var(--toolbarbutton-hover-boxshadow);
   }
 
+  #loop-button[state="slideshow"] > .toolbarbutton-badge-stack {
```

Comment:
Ok, it appears that overall this still isn't right on Mac. Lets move this part of the css into the platform.css for windows & linux only.